boltdb/adaptor_test.go

```diff
@@ -0,0 +1,221 @@
+package boltdb_test
+
+import (
+	"context"
+	"errors"
+
+	"github.com/dogmatiq/dogma"
+	"github.com/dogmatiq/enginekit/fixtures"
+	. "github.com/dogmatiq/projectionkit/boltdb"
+	"github.com/dogmatiq/projectionkit/boltdb/internal"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.etcd.io/bbolt"
+)
+
+var _ = Describe("MessageHandler Adaptor", func() {
+	var (
+		mh *internal.MessageHandlerMock
+		db *internal.TempDB
+	)
+
+	BeforeEach(func() {
+		mh = &internal.MessageHandlerMock{}
+		mh.ConfigureCall = func(c dogma.ProjectionConfigurer) {
+			c.Identity("<projection>", "<key>")
+		}
+		db = internal.NewTempDB()
+	})
+
+	AfterEach(func() {
+		db.Close()
+	})
+
+	Context("HandleEvent method", func() {
+
+		It("executes as expected", func() {
+			ctx := context.TODO()
```

Comment:
Any reason not to use `.Background()` here?

Reply:
Changed to `.Background()` in the latest commits.